cnf-certification-test/accesscontrol/suite.go

```diff
@@ -649,3 +655,110 @@ func Test1337UIDs(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+type ContainerSCC struct {
+	HostDirVolumePlugin bool
+	HostIPC             bool
+	HostNetwork         bool
+	HostPID             bool
+	HostPorts           bool
+	PrivilegeEscalation bool
+	PrivilegedContainer bool
+	RunAsUser           bool
+	Capabilities        string
+}
+
+var (
+	catagory1 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		true,
+		""}
+
+	catagory2 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		false,
+		""}
+
+	catagory3 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		true,
+		"NET_ADMIN, NET_RAW"}
+	catagory4 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		true,
+		"IPC_LOCK, NET_ADMIN, NET_RAW"}
+)
+
+func testContainerSCC(env *provider.TestEnvironment) {
+	var containerSCC ContainerSCC
+	const leetNum = 1337
```

Comment:
leetNum -> istioProxyContainerUID

Reply:
I think `leetNum` is used here because I used it in another place in the code. 😄 It's probably fine to change it.